xslt/shearwater.xslt

```diff
@@ -60,6 +60,17 @@
         <xsl:value-of select="concat(maxTime, ' min')"/>
       </xsl:attribute>
 
+      <xsl:variable name="timeMultiplier">
+        <xsl:choose>
+          <xsl:when test="diveLogRecords/diveLogRecord[1]/currentTime &gt; 100">
```

Comment:
I think that makes sense. if your sample rate is faster than 100ms or slower than 100s, you have other issues as well.